kleinanzeigen_bot/__init__.py

```diff
@@ -404,6 +404,13 @@ def publish_ad(self, ad_file, ad_cfg: dict[str, Any], ad_cfg_orig: dict[str, Any
         if ad_cfg["price_type"] != "GIVE_AWAY":
             self.web_input(By.ID, "pstad-price", ad_cfg["price"])
 
+        #############################
+        # set special properties of category
+        #############################
+        if ad_cfg["special_attributes"]:
```

Comment:
This will raise an exception for ads without the special_attributes. Please add a placeholder entry to https://github.com/Second-Hand-Friends/kleinanzeigen-bot/blob/main/kleinanzeigen_bot/resources/ad_fields.yaml like this:

```yaml
active:
type:
title:
description:
category:
special_attributes: {}
price:
price_type:
shipping_type:
```